packages/@ember/-internals/glimmer/lib/helpers/action.ts

```diff
@@ -278,6 +278,20 @@ export const ACTIONS = new WeakSet();
   @public
 */
 export default internalHelper((args: CapturedArguments): Reference<Function> => {
+  deprecate(
+    `Usage of the \`(action)\` helper is deprecated. Migrate to native functions and function invocation.`,
+    false,
+    {
+      id: 'template-action',
+      url: 'https://deprecations.emberjs.com/v5.x/#toc_template-action',
```

Comment:
We should use the deprecation url in this format:

`https://deprecations.emberjs.com/id/template-action`

This was implemented here https://github.com/ember-learn/deprecation-app/pull/1261 so we can 1) come up with URLs more easily before merging deprecation guides and 2) Have URLs make sense even if we later decide to extend the `until`

Reply:
legit. updated